### DISCUSSION TITLE
System update & upgrade | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1972] [Q&A]
# System update & upgrade | Seeed Studio Wiki

Install and Upgrade OS

https://wiki.seeedstudio.com/recamera_flash_os/

**Comment by s-nazim** [comment]
After running a command, just wanted to upgrade a system by using a local patch

/mnt/system/upgrade.sh start sg2002_reCamera_0.1.4_emmc_ota.zip 

I've got 
Usage: /mnt/system/upgrade.sh start [q|x]

I've also tried with diff options, but still can't, any ideas ? 

My system is  Linux reCamera 5.10.4-tag- #1 PREEMPT Fri Nov 8 04:00:09 UTC 2024 riscv64 GNU/Linux

**Reply by nfs0619** [reply]
Thanks for reaching out! For assistance with your upgrade issue, we recommend contacting our technical support team directly at techsupport@seeed.io. They will be able to provide detailed guidance and solutions tailored to your specific system and situation.
